Automattic/wp-calypso
# Add an InlineSupportLink to a support article in the Blogger importer pane

We need to add more specific support info to the Blogger importer, especially to help people who may not know how to export their site from Blogger. This branch adds an `InlineSupportLink` component to the importer description, which opens the Blogger support article in a modal.

![image](https://user-images.githubusercontent.com/1647564/43517731-9fee2faa-9581-11e8-873a-93d129626ccf.png)

### To Test

* Check out this branch.
* Go to the import page at http://calypso.localhost:3000/settings/import/XXXXX.wordpress.com
* Click the "Start Import" button in the Blogger section.
* In the resulting file upload pane, the description should be worded as above, and the link should open the support article https://en.support.wordpress.com/import/coming-from-blogger/ in a modal window.
